src/PerformanceTests/Tests/Enabled/CSharp_CA2255.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.NetCore.Analyzers.Runtime;
+using PerformanceTests.Utilities;
+using PerfUtilities;
+
+namespace CSharpPerformanceTests.Enabled
+{
+    public class CSharp_CA2255
+    {
+        [IterationSetup]
+        public static void CreateEnvironmentCA2255()
+        {
+            var sources = new List<(string name, string content)>();
+            for (var i = 0; i < Constants.Number_Of_Code_Files; i++)
+            {
+                var name = "TypeName" + i;
+                sources.Add((name, @$"
+class {name}
+{{
+    [System.Runtime.CompilerServices.ModuleInitializer]
+    void Initialize() {{ }}
```

Comment:
isn't it should be static?

Reply:
Seems you wanted to include CS88XY diagnostics, so i making the comment as NIT up to you, but i don't think that is needed, plus it is not clear if one of the  captured 3 diagnostics is actually from this analyzer, i might be wrong but though when there is a compiler error kind of analyzers will not run

Reply:
You're right; thanks! I had created the perf test before adding the logic to enforce public/static, and I failed to revisit this.